backend/README.md

```diff
@@ -40,8 +40,8 @@ If you want to run the smoketests locally, you can use the following commands:
 
 ```bash
    docker-compose up -d
-   docker exec backend_backend_1 pip3 install pytest
-   docker exec backend_backend_1 python3 -m pytest tests/main.py
+   docker exec backend-backend-1 pip3 install pytest
```

Comment:
It uses underscores for me with Docker.

Reply:
It would be probably more portable to replace it with `docker compose exec backend …`.

Reply:
I'm not sure why?
We're using it with `-` in the CI pipeline too and it will fail for me locally with underscore https://github.com/flathub/website/blob/a52252e8a6837ff77d8755eb9ba80818f0f79d1f/.github/workflows/backend_ci.yml#L80

Reply:
Huh, that's weird. Let's keep it as is then.